R/travis.R

```diff
@@ -25,6 +25,9 @@ TravisCI <- R6Class(
     get_commit = function() {
       Sys.getenv("TRAVIS_COMMIT")
     },
+    can_push = function() {
```

Comment:
Do we need a base class implementation, and implementations in other classes?

Reply:
Not sure. We can only deploy on Travis so do implementations for the other classes make sense?

Reply:
There's no reason we can't deploy on AppVeyor, the setup is much more manual though. Pretty sure we also can deploy on other platforms, and also on local.

Reply:
Overlooked Appveyor.

How can we deploy on local?

I'll add it to all.

Reply:
We should assume that local builds can always push, even if `id_rsa` is not set.